server/queue.php

```diff
@@ -1,27 +1,24 @@
 <?php
 	require 'config.php';
 	$conn = $DB_CONN;
+	$displayDate = $_GET['displayDate'];
 
 	// TODO make this handle multiple locations, if necessary
-	$stmt = $conn->prepare('SELECT DISTINCT appointmentId, scheduledTime, firstName, lastName
+	$stmt = $conn->prepare("SELECT DISTINCT appointmentId, scheduledTime, firstName, lastName
 		FROM Appointment
 		JOIN Client ON Appointment.clientId = Client.clientId
-		WHERE (Appointment.scheduledTime >= NOW() AND Appointment.scheduledTime < DATE_ADD(CURDATE(), INTERVAL 1 DAY))
+		WHERE DATE(Appointment.scheduledTime) = '$displayDate'
```

Comment:
correct me if I'm wrong, but this is screaming SQL injection to me. You probably shouldn't directly insert displayDate into this query. You should use the prepared statement to insert a dynamic variable at the execute part